match_file.csv

```diff
@@ -1,26 +1,42 @@
 EXT	FG
 C3_BREAST_EXALLC.csv	C3_BREAST_EXALLC
 C3_PROSTATE_EXALLC.csv	C3_PROSTATE_EXALLC
+C3_PROSTATE_EXALLC_withp_alleles_eur.csv	C3_PROSTATE_EXALLC
```

Comment:
why not adding these variants to C3_PROSTATE_EXALLC.csv? is it bc C3_PROSTATE_EXALLC.csv are review results?

Reply:
yea just left these in so I could see how the results changed.  In the long term they are not needed separately so the other can be removed

Reply:
Same file is ok then.